tests/Integration/Api/Remote/Backups/BackupCompletionNotificationTest.php

```
@@ -0,0 +1,206 @@
+                return $notification->backup->id === $backup->id;
+            }
+        );
+    }
+
+    public function test_email_notification_not_sent_when_config_disabled(): void
+    {
+        Notification::fake();
+        Event::fake([]);  // Clear event fakes to allow real dispatch
+        Config::set('panel.email.send_backup_notification', false);
+
+        [$user, $server] = $this->generateTestAccount();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)